hooks/general_actions.py

```diff
@@ -116,7 +171,15 @@ def execute_action(self, name, params, sg_data):
             assignees.append(app.context.user)
             app.shotgun.update("Task", sg_data["id"], {"task_assignees": assignees})
 
-        elif name == "task_to_ip":        
+        elif name == "add_to_playlist":
+            app.shotgun.update(
```

Comment:
Might be good to throw a debug statement in here.

Reply:
will add!